### PR TITLE
Pretty print deps

### PR DESCRIPTION
**Pull request by fewf**
addresses https://github.com/denoland/deno/issues/1746

still needs tests, and I'd love code review on my Rust, since I'm very new to the language.

### CODE REVIEW

**Comment by fewf** [comment]
One issue that I need to address. Right now if there's a dep that has children and which has already been seen, it prints a `┬` as if it's going to list those children, but because it's been seen, it doesn't print them.

**Comment by fewf** [comment]
Hoping for some help interpreting the test failures. Travis Tests seem to be failing on `./tools/lint.py` but I can't figure out where the lint problem is occurring. And then I'm very unclear about what's going on in the AppVeyor test.

**Comment by ry** [comment]
@fewf Run tools/format.py

File: tests/022_info_flag.out [hunk 1]
```diff
@@ -1,6 +1,14 @@
-local: [WILDCARD]deps/http/127.0.0.1_PORT4545/tests/003_relative_import.ts
+local: [WILDCARD]deps/http/127.0.0.1_PORT4545/tests/019_media_types.ts
 type: TypeScript
 compiled: [WILDCARD].js
 map: [WILDCARD].js.map
-deps: http://127.0.0.1:4545/tests/003_relative_import.ts
-| http://127.0.0.1:4545/tests/subdir/print_hello.ts
+deps:
+http://127.0.0.1:4545/tests/019_media_types.ts
```
Comment (ry) [inline tests/022_info_flag.out]: Nice!

File: src/modules.rs [hunk 2]
```diff
@@ -91,51 +91,71 @@ impl Modules {
...
     }
   }
 }
 
 impl fmt::Display for Deps {
   fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-    for _i in 0..self.depth {
-      write!(f, "| ")?;
+    let mut has_children = false;
+    if let Some(ref deps) = self.deps {
```
Comment (fewf) [inline src/modules.rs]: not sure if there's a more straightforward way of doing basically `obj.array && obj.array.length > 0` here

Comment (ry) [reply src/modules.rs]: What you have is good.

**Pull request merged**